TrenDAP/wwwroot/TypeScript/TrenDAP.tsx

```diff
@@ -60,7 +59,7 @@ const TrenDAP: React.FunctionComponent = (props: {}) => {
             <Application
                 HomePath={homePath} DefaultPath={"WorkSpaces"}
                 Logo={homePath + "Images/trendaplogo.png"}
-                OnSignOut={() => { window.location.href = `${homePath}Logout`; }}>
+                OnSignOut={() => { window.location.href = `/@GSF/Web/Security/Views/Login.cshtml?logout=yes`; }}> { /*need to make sure this page exists on the backend..*/}
```

Comment:
Redirect to `Login` happens in `GSF.Web` so this will need to be changed since we are using Gemstone. Could be significantly more complicated than we want right now.